packages/d3fc-chart/src/cartesian.js

```diff
@@ -0,0 +1,201 @@
+import { select, event } from 'd3-selection';
+import { scaleIdentity } from 'd3-scale';
+import { seriesSvgMulti, seriesCanvasMulti } from '@d3fc/d3fc-series';
```

Comment:
the default plot area for the current cartesian charts is a line series - not sure whether that is any better / worse than a multi?